### PR TITLE
Minutes, plus new working group updates

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Minutes from this week's meeting (2018-10-11) are added.

As the mandate for the new WGs have been agreed, this has been converted from the Google Doc and added to the site.

A new centre column for the front page was written highlighting the additional call for WG convenors.

Stubs for each of the new working groups have been created. As listing all of these as "activities" made the menu item too large, some current activities were also moved into the new "workinggropus" section; the navbar builder now splits between activities and working groups, making this easier to parse by eye. Redirects for pages that moved were made.

An announcement for the upcoming generators workshop was added, as well as an events listing.

A broken link from an old GSOC project was fixed too.

### CODE REVIEW

**Comment by graeme-a-stewart** [comment]
Note that the URL checker fails, but this is a known issue when adding redirects, explained in #310.

File: _layouts/main.html [hunk 1]
```diff
@@ -58,28 +58,30 @@ <h2>Meetings</h2>
...
-    gave a great <a href="https://indico.cern.ch/event/587955/contributions/3012294/">plenary talk</a> 
-    that went beyond the CWP to describe the next steps for HEP Software and Computing.<br>
-	&nbsp;&nbsp; Important common software projects featured highly as well, with a 
-	<a href="https://indico.cern.ch/event/587955/contributions/3012710/">fine talk on common
-	tracking software</a> from Andi Salzburger.
+    The HSF is establishing new working groups with a
+    <a href="/organization/working-group-mandates.html">mandate</a>
+    to pursue the goals from the
+    <a href="/organization/cwp.html">Community White Paper Roadmap</a>
+    last year. We are
```
Comment (eduardo-rodrigues) [inline _layouts/main.html]: Tiny/picky suggestion: groups with <...>mandates</a> to … Roadmap</a> published last year on the <link to arxiv>arXiv</a>.

File: _workinggroups/visualization.md [hunk 2]
```diff
@@ -2,11 +2,12 @@
 title: "Visualization"
 author: "Riccardo Maria Bianchi"
 layout: default
+redirect_from: /activities/training.html
 ---
 
 # Visualization
 
-The Visualization Working Group (WG) gathers experts, develeopers, users and all people from the HEP community interested in interactive data visualization and event displays. The WG has been started in October 2016 and it is convened by Riccardo Maria Bianchi (*University of Pittsburgh*). 
+The Visualization Working Group (WG) gathers experts, develeopers, users and all people from the HEP community interested in interactive data visualization and event displays. The WG has been started in October 2016 and it is convened by Riccardo Maria Bianchi (*University of Pittsburgh*).
```
Comment (eduardo-rodrigues) [inline _workinggroups/visualization.md]: developers

File: events/_posts/2018-10-12-generators.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+layout: event
+startdate: 2018-11-26
+---
+
+The HSF and the LPCC are happy to announce a
+[joint workshop](https://indico.cern.ch/event/751693/) on computing and
+software challenges for event generators.
+
+This event will focus on the challenges for physics event generators that need
+to be faced in HEP in the next decade, may of which were identified in the
```
Comment (eduardo-rodrigues) [inline events/_posts/2018-10-12-generators.md]: many of which

File: _workinggroups/visualization.md (hunk 2 above)
Comment (graeme-a-stewart) [reply _workinggroups/visualization.md]: Ha - not my mistake, but thanks for spotting it (my editor likes to strip trailing whitespace, sometimes that's a good thing!).

**Comment by eduardo-rodrigues** [comment]
I'm merging this now since you want to announce today the 2-week period for consultation on WG conveners. The info this MR brings should be online.

**Pull request merged**